.github/workflows/run-distros.yaml

```diff
@@ -0,0 +1,39 @@
+name: Run Distros Framework
+
+on:
+  push:
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    name: Build and Lint
+    runs-on: ubuntu-20.04
```

Comment:
Do we want to keep these ubuntu-latest for the freemium run time?

Reply:
Any objections?

Reply:
I just got the default one from actions, I can add other

Reply:
doing ubuntu-latest keeps it a free run if we’re under our monthly allowance from github for the org if i recall correctly.

Reply:
sure